apps/native/app/src/screens/applications/applications.tsx

```diff
@@ -76,9 +76,9 @@ export const ApplicationsScreen: NavigationFunctionComponent = ({
   componentId,
 }) => {
   useNavigationOptions(componentId)
-  useOfflineUpdateNavigation(componentId, getRightButtons())
+
   const flatListRef = useRef<FlatList>(null)
-  const [loading, setLoading] = useState(false)
+  const [refetching, setRefetching] = useState(false)
```

Comment:
```suggestion
  const [refreshing, setRefreshing] = useState(false)
```
~🤡 ... I'll show myself out

Reply:
I actually debated my self a lot and changed the property name twice before I decided to use refetching instead of refreshing. The reason I choose refetching is because I thought the name was more descriptive since it is being passed to the useConnectivityIndicator hook and refetch is also the name of the function in the apollo client query.